OData/src/System.Web.OData/OData/Query/Expressions/FilterBinder.cs

```diff
@@ -1779,7 +1779,7 @@ private static Expression CheckIfArgumentsAreNull(Expression[] arguments)
             if (arguments.Any())
```

Comment:
Please add/modify unit test for FilterBinder's change in FilterBinderTest, make sure all unit test is pass.
FYI http://odata.github.io/WebApi/#09-01-unittest-e2etest